components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/sshuseraccesschecktrigger/sshuseraccesschecktrigger.go

```diff
@@ -62,14 +67,19 @@ func (ss *SshUserAccessCheck) Run(config *models.Config) []models.CheckTriggerRe
 	return finalResult
 }
 
-func getSShUserAPIRquest(ip string, sshUser *models.SSHUser) models.SShUserRequest {
-
+func (ss *SshUserAccessCheck) getSShUserAPIRquest(ip string, sshUser *models.SSHUser) models.SShUserRequest {
+	ct, err := ss.FileUtils.ReadFile(filepath.Join(os.Getenv("HOME"), sshUser.PrivateKey))
+	if err != nil {
+		ss.log.Error("Error while opeing the file private file present on the path", err)
```

Comment:
Typo opeing,
Could you modify the line to "Error while opening the file private file present on <actual-path>" ?
Actual-path  is the full path of the file